src/test/java/teammates/logic/core/FeedbackResponseCommentsLogicTest.java

```diff
@@ -30,7 +39,7 @@ protected void prepareTestData() {
 
     @BeforeMethod
     public void refreshTestData() {
-        dataBundle = getTypicalDataBundle();
+        dataBundle = loadDataBundle("/FeedbackResponseCommentsLogicTest.json");
         removeAndRestoreTypicalDataBundle();
```

Comment:
Is this necessary if we are not using `getTypicalDataBundle();`

Reply:
`TypicalDataBundle` does not have the desired data for testing and add new data to it will break many other tests.

Reply:
I mean to suggest that in the case of not using `TypicalDataBundle`, `removeAndRestoreTypicalDataBundle()` would not be necessary. You can check the method declaration here
![Screenshot from 2021-06-11 22-55-21](https://user-images.githubusercontent.com/53283766/121706355-22442400-cb08-11eb-8a68-daa59fa2731d.png)